doc/modules/qualitymetrics/synchrony.rst

```diff
@@ -0,0 +1,49 @@
+Synchrony Metrics (:code:`synchrony`)
+=====================================
+
+Calculation
+-----------
+This function is providing a metric for the presence of synchronous spiking events across multiple spike trains.
+
+The complexity is used to characterize synchronous events within the same spike train and across different spike
+trains. This way synchronous events can be found both in multi-unit and single-unit spike trains.
+Complexity is calculated by counting the number of spikes (i.e. non-empty bins) that occur at the same sample index,
+within and across spike trains.
+
+Synchrony metrics can be computed for different syncrony sizes (>1), defining the number of simultanous spikes to count.
+
+
+
+Expectation and use
+-------------------
+
+A larger value indicates a higher synchrony of the respective spike train with the other spike trains.
+Higher values, especially for high sizes, indicate a higher probability of noisy spikes in spike trains.
```

Comment:
```suggestion
A larger value indicates a higher synchrony of the respective spike train with the other spike trains.
Larger values, especially for larger sizes, indicate a higher probability of noisy spikes in spike trains.
```
High size wouldn't really be used here. So just to make them all agree I would probably switch the values and size to larger. Higher synchrony and higher probability work fine idiomatically.